samples/bookinfo/telemetry/log-entry.yaml

```diff
@@ -23,7 +22,6 @@ apiVersion: config.istio.io/v1alpha2
 kind: handler
 metadata:
   name: newloghandler
-  namespace: istio-system
```

Comment:
what about this one?

Reply:
https://github.com/istio/istio/blob/b6d51c4021da0bf99c598cc33935e5457afb82ec/tests/integration/mixer/telemetry/logs/accesslog_test.go#L49-L57

Here the `ns` is `istio-control` in installer repo. that is why the above error thrown.

Reply:
sorry my comment went on the wrong line.. it looks like there is another Istio system one above.

But I am concerned  this is a sample file, will we break the sample with this?

Reply:
if define namespace in the file, the ci cannot be passed. And I think we should not add `namespace` in yaml file, it is up to the user.

Reply:
Our users look at the samples, we need to make sure they work. I think they need to be in istio-system to work @kyessenov is this right?

If we need to change it for the test we can override it in the test. There is already code to do this for bookinfo anyways so not that big of a deal. We can't break samples to fix tests though

Reply:
The config needs to be either in istio-telemetry namespace or in the app namespace. We will need a follow-up in the docs here https://istio.io/docs/tasks/telemetry/logs/collecting-logs/ to add `-n istio-system` to the shell command.
There is now a test for this sample, so I think it's OK to remove namespace field from config, as long as we update the docs.

Reply:
Thanks @howardjohn and @kyessenov for your review. I have modified the code change to override it in the test instead of modifying the sample.